packages/integration-tests/test/actions.spec.ts

```diff
@@ -65,27 +79,109 @@ const deposit = async (amount, value) => {
   const receipt = await watcher.getL2TransactionReceipt(msgHash)
 }
 
-describe('Messages', async () => {
+const withdraw = async (value) => {
+  const L2Messenger = new Contract(l2MessengerAddress, l2MessengerJSON.interface, l2Wallet)
+  const calldata = simpleStorage.interface.encodeFunctionData('setValue', [value])
+  const l2ToL1Tx = await L2Messenger.sendMessage(
+    simpleStorage.address,
+    calldata,
+    5000000,
+    { gasLimit: 7000000 }
+  )
+  await l2Provider.waitForTransaction(l2ToL1Tx.hash)
+  const count = (await simpleStorage.totalCount()).toString()
+  await sleep(60000)
+}
+
+describe('SimpleStorage', async () => {
+  before(async () => {
+    const web3 = new Web3Provider(
+      ganache.provider({
+        mnemonic: Config.Mnemonic(),
+      })
+    )
+    // TODO: Figure out if Optimism Provider can be used at the same time as JsonRpcProvider
```

Comment:
Don't worry about this for this PR, just using the ethers `JsonRpcProvider` is fine

Reply:
👍 sounds good